### PR TITLE
vm.py Python 3 upgrade (Print statements)

### PR DESCRIPTION
**Pull request by eagleEggs**
Changed the print statements within vm.py to work with Python 3 as well as reflect similar formatting as other modules within the /tools directory

### CODE REVIEW

File: samples/tools/vm.py [hunk 1]
```diff
@@ -37,17 +37,17 @@ def print_vm_info(vm, depth=1, max_depth=10):
...
     if summary.guest is not None:
         ip = summary.guest.ipAddress
         if ip:
-            print "IP         : ", ip
+            print("IP         : {}".format(ip))
     if summary.runtime.question is not None:
-        print "Question  : ", summary.runtime.question.text
-    print ""
+        print("Question  : {}".format(summary.runtime.question.text))
+    print("")
```
Comment (pgbidkar) [inline samples/tools/vm.py]: Hi @eagleEggs Thanks for making this change. Change looks fine
Please can you share testing details you performed for this change. Just to make sure any Syntax error is introduced

Thanks

Comment (eagleEggs) [reply samples/tools/vm.py]: @pgbidkar no problem. I'll add a few details as well...

I found this error when running vcenter_details.py - Which seems to be the only module that imports /tools/vm at least in the format 'from tools import vm'

Before making the change this exception occurred when running vcenter_details.py:

```
Traceback (most recent call last):
  File "vcenter_details.py", line 29, in <module>
    from tools import vm
  File "vm.py", line 40
    print "Name       : ", summary.config.name
                        ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("Name       : ", summary.config.name)?
```

After adding in the parentheses I have ran vcenter_details.py many times with no exceptions.

Comment (eagleEggs) [reply samples/tools/vm.py]: @pgbidkar If you'd like me to do more testing, let me know what you need and I'll get it. I'd just rather this PR not be open for a long time.

Comment (pgbidkar) [reply samples/tools/vm.py]: Hi @eagleEggs Sorry! got occupied with other priority deliverable. Missed your last update.
Merging the change now

Thanks for contributing

Comment (eagleEggs) [reply samples/tools/vm.py]: Thanks @pgbidkar . No problem!

**Pull request merged**